src/main/groovy/de/dkfz/execution/jobs/GenericJobInfo.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright (c) 2016 eilslabs.
+ *
+ * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/Roddy/LICENSE.txt).
+ */
+
+package de.dkfz.execution.jobs;
+
+
+import de.dkfz.eilslabs.tools.conversion.BufferUnit;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by michael on 06.02.15.
+ */
+public class GenericJobInfo {
+
+    private final String jobName;
+    private final String toolID;
+    private final String id;
+    private final Map<String, String> parameters;
+    private List<String> parentJobIDs;
+    private String walltime;
+    private String memory;
+    private BufferUnit memoryBufferUnit;
+    private String cpus;
+    private String nodes;
+    private String queue;
+    private String otherSettings;
```

Comment:
Good question :). Have to look this up and comment it.

tool ID is the used script or tool id. This is an informational class for storing info on executed jobs.